tests/mastertemplate_test.php

```diff
@@ -0,0 +1,71 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_surveypro;
+
+use advanced_testcase;
+use mod_surveypro\mastertemplate;
```

Comment:
You don't need the is "use" statement. It's 100% redundant, because it belongs to the namespace (mod_surveypro) where you already are.

Note this is only for classes straight under the namespace (mod_surveypro/xxxx). If there are more levels (mod_surveypro/form/xxxx for example) then the "use" statement is needed.

But not in this case. That "mod_surveypro" (the current namespace of the file) will be added to all the classes in the files automatically.

Reply:
let me test it
I will push the PR once again as soon as I can.